CONTRIBUTING.md

```diff
@@ -18,12 +18,68 @@
 
 8. If your backport PR(s) get labeled with `bp-conflict`, it means they cannot be automatically be merged. You can help get them merged by openening a PR against the already-existing backport branch (will be named something like `mergify/bp/3.5.x/pr-2512`) with the necessary cleanup changes. The admins will merge your cleanup PR and remove the `bp-conflict` label if appropriate.
 
+---
+
+## Building and Testing
+
+### Dependencies
+
+Chisel uses the [Mill](https://mill-build.org/) build tool.
+You can install it as described on the Chisel [installation instructions](https://www.chisel-lang.org/docs/installation), or just use the bootstrap script in this repository: `./mill`.
+Developers should real the Mill documentation to understand the basic commands and use.
+
+The only real dependency for development is the JDK.
```

Comment:
git is also required for version extraction via mill

Reply:
```suggestion
The main dependencies for development are the JDK and git.
```
TBF, you also need git to clone the repo lol, but worth calling out.

Reply:
Haha, I was bitten this by providing the minimal reproducible dependency by download repo via tar.